.depignore

```diff
@@ -0,0 +1 @@
+openssl
```

Comment:
what is this file for?

Reply:
we check for deps that are unused to purn the dependency graph, that works on a heuristic that the dependency needs to be used in our code. openssl isn't used i our code directly but we need to include it as a dependency for static linking.

To solve the problem of `openssl` being rightfully claimed as a unused dependency but still needed to be a dependency I added the `.depignore` file to ignore dependencies that are in there.